### ISSUE TITLE
build statement in ASP.net? [7:58am]

### ISSUE THREAD

**Issue by alloursjournall** [frankcollins3/ASPIZZA#15]
**attempting to do:**
codespaces looks to be a webapp with a built in editor
![Screen Shot 2023-09-16 at 7 55 28 AM](https://github.com/frankcollins3/ASPIZZA/assets/73137934/4efa78e8-4d5e-4dc5-b29d-21cc2555219d)

**error:**
can't find the code statement from which "build" executes.

**proposed approach:**
we're not in package.json territory. dig some digging have to dig more. 
**possible improvements:**

**Comment by alloursjournall** [comment]
AI ADVICE:
![Screen Shot 2023-09-16 at 8 00 24 AM](https://github.com/frankcollins3/ASPIZZA/assets/73137934/87c59382-748b-493b-8337-14167a7e511a)
codespace: ContosoPizza.csproj (just like chatGPT with the same syntax)
![Screen Shot 2023-09-16 at 8 00 38 AM](https://github.com/frankcollins3/ASPIZZA/assets/73137934/154aecd4-1a01-479c-96c7-9dcfc0a83a21)
[8:01am]

**Comment by alloursjournall** [comment]
👎 throwing 2% chance out there
![Screen Shot 2023-09-16 at 8 02 56 AM](https://github.com/frankcollins3/ASPIZZA/assets/73137934/ec30f10d-521e-4e9b-9412-cbe9c646aef5)
[8:03am]

**Comment by alloursjournall** [comment]
{duncecap.png} () => dotnet build lol
[8:04am]

**Comment by alloursjournall** [comment]
bookmarking that i'm asking the silliest questions right now.
 should now to try dotnet run but don't want to do anything without knowing first in this [codespace]
![Screen Shot 2023-09-16 at 8 05 34 AM](https://github.com/frankcollins3/ASPIZZA/assets/73137934/a835c403-cdad-4802-b401-024ecd69f527)

![Screen Shot 2023-09-16 at 8 06 56 AM](https://github.com/frankcollins3/ASPIZZA/assets/73137934/028bf996-b46e-4247-a15b-cda23a4af1d2)

![Screen Shot 2023-09-16 at 8 07 01 AM](https://github.com/frankcollins3/ASPIZZA/assets/73137934/7594f835-21f7-4d7c-a413-9086c364bfef)
[8:07am]

**Comment by alloursjournall** [comment]
dont feel so dumb now:
![Screen Shot 2023-09-16 at 8 09 10 AM](https://github.com/frankcollins3/ASPIZZA/assets/73137934/684be6f1-d208-4525-a8e8-c094f0ffb515)
[8:08am]

**Comment by alloursjournall** [comment]
👍  cd ContosoPizza -> dotnet run.
![Screen Shot 2023-09-16 at 8 10 03 AM](https://github.com/frankcollins3/ASPIZZA/assets/73137934/68fbe09a-642b-4bac-89a4-d7e205feb2cb)

// sidenote:
when we were making the app by going through the walkthrough, it did strike me as unusual when we proceeded to apparently deal with a SQLite/Entity link that was never set up.
I chalked it up to "SQLite" being quite "Lite" indeed. Codespace makes so much sense that you hook up to storage.

last night I planned to move on from doing this ASP.net/razor code to go solely to ASP.net core /React front end
I chalked that up to trying to code in ASP.net on an old mac and having cross platform issues?
[8:12am]
